tests/classify_test.py

```diff
@@ -130,6 +130,22 @@ def test_application_directories(in_tmpdir, no_empty_path):
     assert ret is ImportType.APPLICATION
 
 
+@pytest.mark.xfail(
+    sys.platform.startswith('linux'),
+    reason='Expected fail for case-sensitive file system',
+)
+@pytest.mark.xfail(
+    sys.platform.startswith('darwin'),
```

Comment:
better would be `sys.platform != 'win32'`

Reply:
Done: https://github.com/asottile/aspy.refactor_imports/pull/53/commits/3685fe2020e9a31473301bd0aaf111fa53a7b0a7